index.html

```diff
@@ -1212,10 +1214,10 @@ <h3>Multikey</h3>
 
             <p>
 In the example above, the `secretKeyMultibase` value starts with the letter `z`,
-which is the [[?MULTIBASE]] header that conveys that the binary data is
-base58-encoded using the Bitcoin base-encoding alphabet. The decoded binary data
-[[?MULTICODEC]] header is `0x1300`, which specifies that the remaining data
-is a 32-byte raw Ed25519 private key.
+which is the <a href="#multibase-0">Multibase</a> header that conveys that the
+binary data is base-58-btc-encoded using the Bitcoin base-encoding alphabet. The
+decoded binary data header is `0x8026`, which specifies that the
+remaining data is a 32-byte raw Ed25519 private key.
```

Comment:
How does 0x8026 specify the format of the data?  This needs to be defined so that implementers can do the same thing.

Reply:
You are referring to an example. Examples don't normatively define things in the specification. The normative definition of that value is in the EdDSA Cryptosuite specification:

https://w3c.github.io/vc-di-eddsa/#multikey

`0x8026` is the Multicodec header for a Ed25519 private key (which is what the text above says). This is normatively defined in the EdDSA Cryptosuite specification and it's done in a way where it's clear what the entire field should be. If you disagree, please provide concrete text to address your concern.